client/src/components/organisms/TimeLine/TimeLine.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import './TimeLine.css';
+
+import IcoMoon from 'react-icomoon';
+
+const TimeLine = () => {
+  return (
+    <Container>
+      <ul className="cbp_tmtimeline">
+        <li>
+          <time className="cbp_tmtime" dateTime="2013-04-10 18:30">
+            <span>4/10/13</span> <span>18:30</span>
+          </time>
+          {/* <div className="cbp_tmicon cbp_tmicon-phone" /> */}
+          <IcoMoon icon="phone"></IcoMoon>
+          <div className="cbp_tmlabel">
+            <h2>Ricebean black-eyed pea</h2>
+            <p>
+              Winter purslane courgette pumpkin quandong komatsuna fennel green
```

Comment:
All these hard-coded values will be replaced by the axios fetching in some other PR?